dist/form.js

```diff
@@ -14,32 +14,4 @@ form.addEventListener('submit', (e) => {
     feedBack.textContent = `${fullName} is  inValid`;
     feedBack.style.borderColor = '#fc0b03';
```

Comment:
The ```form.js``` has been linked to ```index.html``` as requested, and I will like to clarify that the  
Regex ``` const fullNamePattern = /^[A-Za-z]{6,20}$/;``` on line 11 doesn't stop you from submitting emails with uppercase if that is what you thought it will do. 😄